### DISCUSSION TITLE
[Q] arraybuffer issue

### DISCUSSION THREAD

**Discussion by softboy99** [exceljs/exceljs#2446] [Q&A]
workbook.eachsheet call back can not be called:

var req = new XMLHttpRequest();
		req.open("GET", fileUrl, true);
		req.responseType = "arraybuffer";
		req.onload = function(e) {
			// load from buffer
			var workbook = new ExcelJS.Workbook();
                               var data = req.response;
				workbook.xlsx.load(data).then(workbook => {
				  console.log(workbook, 'workbook instance')
				  workbook.eachSheet((sheet, id) => {
					console.log(sheet);
				  })
				})
		}
		req.send();
	}

**Comment by softboy99** [comment]
Hi,
The above code cannot load from arraybuffer. Is there any error?

**Comment by softboy99** [comment]
Hi,
Is it any incorrect code in the above, why workbook.xlsx.load(data) failed?

**Comment by softboy99** [comment]
Hi,
why no response?

**Comment by Alanscut** [comment]
@softboy99 Please see issue #1168 , the example in this issue can handle ArrayBuffer data normally.
